Plugin/CleverTapUnity/CleverTapUnitySDK/Common/CleverTapPlatformBindings.cs

```diff
@@ -0,0 +1,303 @@
+﻿using CleverTap.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace CleverTapUnitySDK.Common {
+    public abstract class CleverTapPlatformBindings {
+        public const string Version = "2.4.1";
```

Comment:
We can move this version to constants. i.e `CleverTapUnitySDK/Constants/CleverTapVersion.cs`. We might use it to bump the version from build scripts later

Reply:
A New CleverTapVersion class has been added and used instead of a local constant